cognite/client/data_classes/datapoints.py

```diff
@@ -213,16 +213,41 @@ def _ts_info(self) -> dict[str, Any]:
             "granularity": self.granularity,
         }
 
+    @typing.no_type_check
     @classmethod
-    def _load(  # type: ignore [override]
+    def load(  # type: ignore [no-untyped-def, unused-ignore]
```

Comment:
The ignore is unused since you have added `@typing.no_type_check`, so you may remove both if I understand correctly:
```suggestion
    def load(
```